client/src/components/brushableHistogram/index.js

```diff
@@ -38,12 +38,11 @@ class HistogramBrush extends React.Component {
     return varData.col(field);
   }
 
-  calcHistogramCache = memoize((world, field) => {
+  calcHistogramCache = memoize((col, field) => {
```

Comment:
Was this optimization specific to this PR, or unrelated?

Reply:
as part of debugging this, I was trying to understand why we were destroying and recreating the SVG brush/axis layer with any change to world.  Found this bad memoization (which I think was originally my bug).

Strictly speaking we could omit this and still fix this bug, but it was quite simple and a nice perf fix, so I didn't delete the code from the PR.

Reply:
I had a feeling - great find.